contracts/libraries/PublishingLogic.sol

```diff
@@ -89,12 +89,14 @@ library PublishingLogic {
             _profile.followModule = followModule;
         }
```

Comment:
if there is no change in the `followModule`, should we call `_initFollowModule` too?

Reply:
I believe that yes, because it could be using different `followModuleData`, basically re-initializing it with other parameters

Reply:
Alan is right, we need to call it anyway.